packages/web-client/src/generated/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.5.0-SNAPSHOT
+6.5.0-SNAPSHOT
```

Comment:
Still don't understand what's wrong with the versioning of the generated api client... or is this really an accidental downgrade?